packages/fxa-admin-server/src/config.ts

```diff
@@ -54,7 +53,7 @@ const conf = convict({
     env: 'NODE_ENV',
     format: ['development', 'test', 'stage', 'production'],
   },
-  logging: {
+  log: {
```

Comment:
At the moment the shared LoggerModule relies on the key being `log` rather than requiring the config to be passed in. If we tire of this config requirement we could add manual configuration for LoggerModule as well.